config/pickit.ini

```diff
@@ -1106,6 +1132,7 @@ uniq_weapon_ribcracker=1, ethereal
 uniq_hand_of_blessed_light=1,, ethereal
 uniq_heavens_light=1,, ethereal
 uniq_rune_master=1, ethereal
+uniq_weapon_grandfather=1
 
```

Comment:
please make all of the new items you added have the first field be `=0` by default, users can then change them to 1 if desired.